docs/reference/support-policy.md

```diff
@@ -86,6 +86,7 @@ The table below describes recent versions of Stencil and the version of TypeScri
 
 | Stencil Version | TypeScript Version |
 |:---------------:|:------------------:|
+|     v4.13.0     |       v5.4.0       |
```

Comment:
I think we only need to change this file - historically we don't backfill the version support doc. Can we reset the other versioned docs to their previous state?

Reply:
I've done it with the last TS update but went back and removed these references again.